displayio/colorconverter.py

```diff
@@ -122,3 +135,16 @@ def dither(self, value):
         if not isinstance(value, bool):
             raise ValueError("Value should be boolean")
         self._dither = value
+
+    @property
+    def rgba(self):
```

Comment:
Why are you adding this? Doing so means people may not be able to move their code back to CircuitPython.

Reply:
This was to specify that it should return the value in rgba format and is really for internal use. The other option is that I could just directly set the private variable from the Tilegrid class, which is fine with me.

Reply:
Ok, I removed the property and am just accessing the variable directly so users shouldn't make incompatible code now.